bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/AsyncExecutor.java

```diff
@@ -262,6 +306,21 @@ public void flush() throws IOException {
     LOG.trace("Done flushing");
   }
 
+  public ListenableFuture<MutateRowsResponse> mutateRowsAsync(BulkMutation bulkMutation)
```

Comment:
Should have a method comment like (most of) the rest of the public methods in this class.